TTDQueries/tutorial-instructions.md

```diff
@@ -0,0 +1,243 @@
+# Time Travel Debugging and Queries
+
+This tutorial demonstrates how to debug C++ code using a [Time Travel Debugging]
+recording. This example focuses on the use of Queries to find information about
+the execution of the code in question.
+
+## Prerequisites
+
+Note that this tutorial only applies to Windows 10 environments.
+Download the following software in order to follow along:
+
+* WindbgNext from the Windows Store

```

Comment:
WinDbg Preview

Reply:
Thank you :) I searched and made sure there are no others